spec/models/form_submission_spec.rb

```diff
@@ -40,7 +40,7 @@
       it 'returns benefits_intake_id from the form submission' do
         result = FormSubmission.with_latest_benefits_intake_uuid(user_account).with_form_types(['FORM-C']).first
 
-        expect(result.benefits_intake_uuid).to eq(@fsc.benefits_intake_uuid)
+        expect(result.benefits_intake_uuid).to be_nil
```

Comment:
is this meant to be nil now?  if so, do we need to scan the code for any place that might expect this to exist now?

Reply:
Oh good call! This should be `nil` but the `it` block doesn't reflect that.